src/components/ObjektDisplay.vue

```diff
@@ -1,29 +1,59 @@
 <script setup lang="ts">
-    const objektsQuery = `
-        query MyQuery 
-            {collections(limit: 15){
-                front
-                id
-            }
+    import { ref, watch } from "vue"
+    const props = defineProps({
+        objektClass : String,
+        objektSeason : String,
+        objektGroup : String,
+        objektArtist : String
+    })
+    
+    const imageList = ref<any[]>([])
+    
+    const recalculateImageList = async() => {
+        const queryFilters = {
+            class_eq : props.objektClass,
```

Comment:
You usually don't put a space between both sides of the colon (`:`), when defining an object. I'd suggest getting and setting up `prettier`. Installing it via npm is one thing. You need to set it up with your IDE as well. I suggest to trigger it automatically on save.